examples/onprem_demo/README.md

```diff
@@ -0,0 +1,70 @@
+- A service account with Project Creator privileges.  Steps for provisioning this are included below, but those steps assume you have Organization Admin privileges.
+- Terraform v0.11.13 with provider.google v2.6.0 (the only versions I've tested; run `terraform version` to see what you've got).
+
+## Instructions
+
+0. Set up the service account needed to run the Terraform code.
+    - In an existing project, create the service account.  Copy and paste its email address (the big, long, `gserviceaccount.com` address) to a scratch pad.  Download its .json secret key file into the `onprem_project/` and also copy it into the `vpc_sc_project` folders.  Be **extremely careful** with this file; delete it when you're done with this example.
+    - Go to the [IAM admin](https://console.cloud.google.com/iam-admin/iam) section of your *organization.*  (You should see your domain name at the top of the screen, not an individual project name).  Click Add.  In the "New Members" box, paste the email address of the service account you created.  Under the Roles dialog, search for "Project Creator."  Grant that role by clicking Save.  Your service account can now create projects and act as a project owner for those projects.
+    - Go to the Billing section of the Cloud Console.  Select the billing account you want to charge for the projects you create.  (If you're using your credit card, the default limit is 5 projects mapped to a billing account, in case project creation fails for you in the steps below).  On the right side of the screen, there's a "Permissions" section for your billing account.  Click "Add members," and paste the email address of the service account you created.  Grant the service account Billing Admin privileges, so that it can map the projects it creates to your billing account.  **Note the billing account ID** (the alphanumeric text separated by hyphens), as we'll use it below.
+
+1. Create the Terraform variables file.
+    - In your favorite text editor, edit and save the following variables in `examples/onprem_demo/terraform.tfvars`:
+        - `vpc_sc_project_id`: the unique ID for the VPC Service Control project that is going to be created.  Use lowercase letters and a 6 digit random number only (no spaces).
+        - `onprem_project_id`: the unique ID for the on-prem project that is going to be created.  Use lowercase letters and a 6 digit random number only (no spaces).
+        - `credentials_path`: the name of the service account secret key file you downloaded.  Relative path is fine if you saved it in the `onprem_project` directory.
+        - `organization_id`: run `gcloud organizations list` to get this **number**.
+        - `billing_account_id`: the billing account that the project will charge to.
+        - `region`: region where the GCP resources will be created. 
+        - `vpn_shared_secret`: any random text string can go here.
+        - `access_policy_name`: Get this value by running the following two commands:
```

Comment:
As per this [comment](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/pull/4#discussion_r301367739) you should add a step to create the policy as a user may not have used VPC SC before. Or maybe a conditional step like "if `gcloud access-context-manager policies list --organization <Organization ID> ` returns nothing, create the policy"

Reply:
Done.